myria3d/pctl/points_pre_transform/lidar_hd_norgb.py

```diff
@@ -0,0 +1,46 @@
+# function to turn points loaded via pdal into a pyg Data object, with additional channels
```

Comment:
this needs tests as well, that can be added in https://github.com/IGNF/myria3d/tree/main/tests/myria3d/pctl/transforms